src/Parsers/Kusto/KustoFunctions/KQLCastingFunctions.cpp

```diff
@@ -4,9 +4,17 @@
 #include <Parsers/Kusto/KustoFunctions/KQLFunctionFactory.h>
 
 #include <format>
-#include <regex>
 #include <Poco/String.h>
 
+#ifdef __clang__
```

Comment:
May be we should wrap that staff in some our header? It looks excessive.

Reply:
Agree, I'll check.